### PR TITLE
Render A4A ads in a sample article

### PR DESCRIPTION
**Pull request by sebastianbenz**
* different previews can be configured:
    `"preview": "default"`
    `"preview": "a4a"`
* new gulp task to serve API
* migrated existing samples to new preview syntax (the old way
  with `preview: true` still works)
* A4A ads are embedded in preview via iframe -> needs to be deployed to
  API app engine instance

### CODE REVIEW

**Comment by sebastianbenz** [comment]
//cc @carlosfuertes

File: src/60_Ads_for_AMP_(A4A)/A4A_Carousel.html [hunk 1]
```diff
@@ -39,35 +40,29 @@
...
-         href="<%host%>/ads_for_amp_a4a/a4a_carousel/">
-        <amp-img src="/img/a4a_slide1.png"
-                 width=300
-                 height=250
-                 alt="slide 1"/>
-      </a>
-    </div>
+    <a target="_blank"
+       href="<%host%>/ads_for_amp_a4a/a4a_carousel/">
+       <amp-img src="<%host%>/img/a4a_slide1.png"
```
Comment (sebastianbenz) [inline src/60_Ads_for_AMP_(A4A)/A4A_Carousel.html]: @carlosfuertes FYI I had to make all image paths absolute as the ads are served in the preview from a different domain. Is that OK from you side or does it send the wrong signal?

File: src/60_Ads_for_AMP_(A4A)/A4A_Carousel.html [hunk 2]
```diff
@@ -39,35 +40,29 @@
   <amp-carousel width="300"
                 height="250"
                 type="slides">
```
Comment (sebastianbenz) [inline src/60_Ads_for_AMP_(A4A)/A4A_Carousel.html]: @carlosfuertes FYI I removed the `div` in your carousel sample as it is not needed.

**Comment by sebastianbenz** [comment]
@kul3r4 PTAL

This is what it looks like: 

* [A4A Banner Ad](https://amp-by-example-staging.appspot.com/ads_for_amp_a4a/a4a_banner_ad/preview/)
* [A4A Carousel](https://amp-by-example-staging.appspot.com/ads_for_amp_a4a/a4a_carousel/preview/)

**Comment by kul3r4** [comment]
LGTM

**Pull request merged**